code/backtracking/knight_tour/knight_tour.go

```diff
@@ -0,0 +1,87 @@
+// Part of Cosmos by OpenGenus Foundation
+package main
+
+import "fmt"
+
+/* Expect Result
+
+ 0  59  38  33  30  17   8  63
+37  34  31  60   9  62  29  16
+58   1  36  39  32  27  18   7
+35  48  41  26  61  10  15  28
+42  57   2  49  40  23   6  19
+47  50  45  54  25  20  11  14
+56  43  52   3  22  13  24   5
+51  46  55  44  53   4  21  12
+*/
+type Move struct {
+	X int
+	Y int
+}
+
+var boardSize int
+
+func isSafe(nextX, nextY int) bool {
+	if nextX >= 0 && nextX < boardSize && nextY >= 0 && nextY < boardSize {
```

Comment:
This function could be simplified to one line:

```go
return nextX >= 0 && nextX < boardSize && nextY >= 0 && nextY < boardSize
```

Reply:
Thanks.